### ISSUE TITLE
Add local facilities to the geowidget - CHW

### ISSUE THREAD

**Issue by msschroeder** [opensrp/opensrp-client-chw-emergency-transport#3]
- [x] Area facilities should appear on the geowidget
- [x] The GeoJSON will be provided
- [x] For the cardview for the facility:
- Facility Name, Facility Type, Ownership, Basic Services Provided (Yes/No), CEmONC (Yes/No)

- [ ] There will be a different color for facilities with CEmONC = Yes (#11d472) and facilities with CEmONC = no (#116cd4)

@rowo  What colors would you recommend for each of these types of structures? Should we only use different point colors or also different icons for either each type of point (household vs. community responder vs. facility) or for each type of facility?

Possible colors - #15cfd4, #5c44ef, #c81cdd, #88F931, #0004FF, #72FFFB, #B0FF72, #FFCE00, #FFA2E3

Link to GeoJSON: https://onaio.slack.com/archives/CCBBGFR2B/p1588789220014500

**Comment by msschroeder** [comment]
@githengi Couple of questions:
1.  In what formats, do the GPS need to be saved? i.e. in separate columns, or a combined column? If, a combined column, which should come first: latitude or longitude
2. I assume the location hierarchies have to align with those on OpenMRS? Meaning the name on OpenMRS and in the file have to be the same? If so, are there any additional field that should be mapped in the same process (like adding a code)?

**Comment by githengi** [comment]
The GPS can be saved on any format, however they should be saved a Features so that rendering on the map there is not conversion of to Features.

The Location entity can be used and points saved on the structures tables

**Comment by githengi** [comment]
Yes the locations on the file should match the OpenMRS hierarchies. If the OpenMRS location id is provided that would be useful to filtering which a locations a user has access to. 

On Reveal all locations are tagged with the OpenMRS Id

**Comment by githengi** [comment]
@msschroeder could you provide the sample CSV. 
This will need posting the contents to the Location API as features(structures).

@msschroeder also scope a criteria to identify which facilities will be shown for user.

**Comment by githengi** [comment]
LOE 3-5 days for 1 FTE

**Comment by msschroeder** [comment]
We are waiting on the client to provide this.
